src/content/doc-surrealql/statements/define/access/bearer.mdx

```diff
@@ -0,0 +1,217 @@
+---
+sidebar_position: 10
+sidebar_label: BEARER 
+title: DEFINE ACCESS ... TYPE BEARER statement | SurrealQL
+description: A bearer access method allows accessing SurrealDB using a bearer key.
+---
+
+import Since from '@components/shared/Since.astro'
+
+# `DEFINE ACCESS ... TYPE BEARER`
+
+<Since v="v2.2.0" />
+
+> [!CAUTION]
+> Currently, the `BEARER` access method is an experimental feature intended to be used for validating its suitability and security. As such, it may be subject to breaking changes and may present unidentified security issues. Do not rely on this feature in production applications. To enable this and other experimental features related to bearer access, set the `SURREAL_EXPERIMENTAL_BEARER_ACCESS` environment variable to `true`.
```

Comment:
```suggestion
> Currently, the `BEARER` access method is an experimental feature intended to be used for validating its suitability and security. As such, it may be subject to breaking changes and may present unidentified security issues. Do not rely on this feature in production applications. To enable this and other experimental features related to bearer access, set the `SURREAL_EXPERIMENTAL_BEARER_ACCESS` [environment variable](/docs/surrealdb/cli/env) to `true`.
```